dynawo/sources/Solvers/SolverSIM/DYNSolverSIM.h

```diff
@@ -282,6 +282,7 @@ class SolverSIM : public Solver::Impl {
   std::vector<double> ySave_;  ///< values of state variables before step
   std::vector<double> zSave_;  ///< values of discrete variables before step
   std::vector<state_g> gSave_;  ///< values of roots before step
+  bool skipScaling_;  ///< flag used to skip algebraic residues evaluation after a convergence or a mode
```

Comment:
I don't really like skipScaling_ as it could be misleading in the sense that we do scaling. What we don't do is evaluateAlgebraicResiduals.

Reply:
done